ldk/javascript/src/types/oliveHelps/index.d.ts

```diff
@@ -290,7 +290,7 @@ declare namespace OliveHelps {
   interface Whisper {
     id: string;
     close: Readable<undefined>;
-    update(whisper: UpdateWhisper, cb?: (err: Error) => void): void 
+    update(whisper: UpdateWhisper, cb?: (err: Error) => void): void
```

Comment:
Are you missing semicolon?

Reply:
I think this is where VSCode knocked off a trailing space. I'll check to be sure though.